index.js

```diff
@@ -25,16 +33,44 @@ function replaceExtensions(extensionsRegex, name) {
   return name;
 }
 
+function pluginCanBeParallelized(plugin) {
```

Comment:
ok - I moved this stuff to a separate file as well to keep index.js a little cleaner